wireless/bluetooth/Kconfig

```diff
@@ -165,6 +165,16 @@ config BLUETOOTH_BUFFER_IRQRESERVE
 		interrupt level.  This setting only needs to be non-zero if your
 		low-level Bluetooth driver needs to do such allocations.
 
+config BLUETOOTH_CNTRL_HOST_FLOW_ENABLE
+	bool "Enable Controller to Host Flow Control"
+	default n if ARCH_CHIP_ESP32
```

Comment:
@robertc2000 actually we don't include arch specific rules here. Case you need to control it this way, it is possible to create an intermediate symbol (i.e. BLE_HOST_HAS_FLOW_CONTROL) and in the arch/esp32 you enable the BLE_HOST_HAS_FLOW_CONTROL